packages/api/core/src/api/init.ts

```diff
@@ -118,17 +120,32 @@ export default async ({ dir = process.cwd(), interactive = false, copyCIFiles =
                   }
                   await installDepList(dir, templateModule.devDependencies || [], DepType.DEV);
                 },
+                exitOnError: false,
```

Comment:
`exitOnError` default is `false`. If we decide to make an explicit declaration, do we also need to change other places?

Perhaps we could submit another PR to have explicit declarations in all places?

Reply:
> exitOnError default is false. If we decide to make an explicit declaration, do we also need to change other places?

Yeah, that could be a good idea 👍

I resorted to explicit declarations here because having a parent declaration of `exitOnError: false`, while one of its children's tasks has a declaration of `exitOnError: true` is confusing 😆  And it's not clear if errors in the child will cause a process to exit (it does, but it's not documented and might be hard to remember).

Reply:
Alright, let's do that. We can handle explicitly declaring exitOnError in another PR.